process_functions.py

```
@@ -1,3 +1,855 @@
+                try:
+                    docs[abs_path_pdf].close()
+                except Exception as ex:
+                    None    
+                    
+
+def find_similar(bytes_from_image):
+    image_match_dir = os.path.join(utilities_general.get_application_path(), 'image_match')
+    image_match_exe_abs = os.path.join(image_match_dir, 'image_match.exe')
+    tempfile = os.path.join(utilities_general.get_application_path(), "temp_im.png")
+    #print(tempfile)
+    f = open(tempfile, 'wb')
```

Ruff

Use a context manager for opening files

(SIM115)